### PR TITLE
updating Mailjet code samples for Send API v3.1

### PR DESCRIPTION
**Pull request by adamyanliev**
Mailjet Send API v3.1 is being released, so updating the code samples to match the new Send API v3.1 configuration. Reference can be found [here](https://dev.mailjet.com/beta/).

### CODE REVIEW

File: appengine/standard/mailjet/app.php [hunk 1]
```diff
@@ -40,31 +40,34 @@
...
-        'FromEmail' => "test@example.com",
-        'FromName' => "Testing Mailjet",
-        'Subject' => "Your email flight plan!",
-        'Text-part' => "Dear passenger, welcome to Mailjet! May the delivery force be with you!",
-        'Html-part' => "<h3>Dear passenger, welcome to Mailjet!</h3><br/>May the delivery force be with you!",
-        'Recipients' => [
-            [
-                'Email' => $recipient,
-            ]
+    'Messages' => [
```
Comment (bshaffer) [inline appengine/standard/mailjet/app.php]: This is the wrong indentation, everything after body needs another indent.

File: appengine/flexible/mailjet/app.php [hunk 2]
```diff
@@ -40,31 +40,34 @@
...
-        'FromEmail' => "test@example.com",
-        'FromName' => "Testing Mailjet",
-        'Subject' => "Your email flight plan!",
-        'Text-part' => "Dear passenger, welcome to Mailjet! May the delivery force be with you!",
-        'Html-part' => "<h3>Dear passenger, welcome to Mailjet!</h3><br/>May the delivery force be with you!",
-        'Recipients' => [
-            [
-                'Email' => $recipient,
-            ]
+    'Messages' => [
```
Comment (bshaffer) [inline appengine/flexible/mailjet/app.php]: Same here

**Comment by adamyanliev** [comment]
hi @bshaffer, thanks for the comments. I fixed the indentation, it should be ok now. Also, yes, the samples were manually deployed, tested and working.

**Comment by adamyanliev** [comment]
@bshaffer the Travis CI build fails now, seems like the PHP CS Fixer needs to be updated on your end?

**Comment by bshaffer** [comment]
@adamyanliev working on it... https://github.com/GoogleCloudPlatform/php-tools/pull/21

**Comment by bshaffer** [comment]
Before I merge this can you mention the reasons for opting into Send API v3?

**Comment by bshaffer** [comment]
It appears the client library does not default to v3 (we explicitly set the version), so we are opting into V3. Why?

**Comment by adamyanliev** [comment]
@bshaffer v3 is indeed the default - see the below: 
https://github.com/mailjet/mailjet-apiv3-php/blob/master/src/Mailjet/Client.php#L35
https://github.com/mailjet/mailjet-apiv3-php/blob/master/src/Mailjet/Config.php#L21

**Comment by bshaffer** [comment]
If it's the default why are we specifying `['version' => 'v3']` when making the call?

**Comment by adamyanliev** [comment]
@bshaffer sorry, our mistake, version call should be 3.1. I updated the files to match, thank you for pointing it out.

**Comment by bshaffer** [comment]
This is now returning 400 Bad Request. When I revert this PR, it works as expected.

**Pull request merged**